src/engine/StoryState.ts

```diff
@@ -536,23 +571,68 @@ export class StoryState {
       );
     }
 
-    this.callStack.SetJsonToken(jObject["callstackThreads"], this.story);
+    let flowsObj = jObject["flows"];
+    if (flowsObj != null) {
+      let flowsObjDict = flowsObj as Record<string, any>;
+
+      // Single default flow
+      if (Object.keys(flowsObjDict).length == 1) {
```

Comment:
There are 2 double equal compares on lengths around here, I think they could be strict equals like everywhere else?